package.json

```diff
@@ -86,15 +86,15 @@
     "@types/stats-lite": "^2.2.0",
     "@uniswap/default-token-list": "^11.13.0",
     "@uniswap/permit2-sdk": "^1.3.0",
-    "@uniswap/router-sdk": "^1.14.0",
-    "@uniswap/sdk-core": "^5.9.0",
-    "@uniswap/smart-order-router": "4.8.1",
+    "@uniswap/router-sdk": "^1.15.0",
+    "@uniswap/sdk-core": "^6.0.0",
+    "@uniswap/smart-order-router": "file:/tmp/uniswap-smart-order-router-4.8.2.tgz",
```

Comment:
update once https://github.com/Uniswap/smart-order-router/pull/766 is checked in